components/raftstore-v2/src/operation/life.rs

```diff
@@ -126,18 +132,23 @@ pub struct AbnormalPeerContext {
     pending_peers: Vec<(u64, Instant)>,
     /// A inaccurate cache about which peer is marked as down.
     down_peers: Vec<u64>,
+    // disk full peer set.
+    disk_full_peers: DiskFullPeers,
+    // show whether an already disk full TiKV appears in the potential majority set.
+    dangerous_majority_set: bool,
 }
 
 impl AbnormalPeerContext {
     #[inline]
     pub fn is_empty(&self) -> bool {
-        self.pending_peers.is_empty() && self.down_peers.is_empty()
+        self.pending_peers.is_empty() && self.down_peers.is_empty() /* && self.disk_full_peers.is_empty() */
     }
 
     #[inline]
     pub fn reset(&mut self) {
         self.pending_peers.clear();
         self.down_peers.clear();
+        // self.disk_full_peers.clear();
```

Comment:
is it needed?

Reply:
Missed it. It's redundant here, as the disk_full_peers will be refreshed at each tick if the disk_usage is updated.